fixtures/worker-app/tests/index.test.ts

```diff
@@ -76,4 +76,13 @@ describe("'wrangler dev' correctly renders pages", () => {
 		const text = await response.text();
 		expect(text).toMatch(/[0-9a-f]{16}/); // 8 hex bytes
 	});
+
+	it("passes through URL unchanged", async ({ expect }) => {
+		const url = `http://${ip}:${port}//thing`;
```

Comment:
It wasn't a problem in the linked issue, but it might be worth adding a query string and a fragment here, just in case